docs/source/examples.rst

```diff
@@ -214,6 +214,55 @@ Animations
             self.play(square.animate.scale(0.3))
             self.play(square.animate.rotate(0.4))
 
+.. manim:: MovingAngle
+    :ref_modules: manim.mobject.geometry.Angle
+    :ref_functions: Mobject.rotate
```

Comment:
This reference seems to be broken.
Not only in this pr, but in all reference fields in the example gallery

Reply:
If the referenced module, class, or function doesn't have a docstring, then there is nothing to link to, making the link appear broken. `Mobject.rotate` doesn't have a docstring.

Also, `manim.mobject.geometry.Angle` isn't a module -- you want to reference the class in this case.